qark/report.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+
+from os import path
+
+from jinja2 import Environment, PackageLoader, select_autoescape, Template
+
+from qark.vulnerability import (Vulnerability, Severity)
+
+DEFAULT_REPORT_PATH = '{}/report'.format(path.abspath(path.dirname(path.realpath(__file__)) + '/..'))
```

Comment:
Use `path.join` to combine these instead of string formatting. `os.path.join` will pick the correct directory separator.

Reply:
`abspath` uses `join` under the hood in most cases. 
https://docs.python.org/2/library/os.path.html#os.path.abspath

But agreed that the rest of them should use `join` explicitly so will change this one as well.

Reply:
Fixed.